app/models/pawn.rb

```diff
@@ -1,9 +1,24 @@
 class Pawn < ChessPiece
-  def display
-    if color == "black"
-      return "&#9823;"
-    else
-      return "&#9817;"
-    end
+
+  def valid_move?(board, stop)
+     valid_move_by_two?(stop) || 
+     (valid_direction?(start, stop) && spaces_between(start, stop) == 1 && !is_space_occupied?(board, stop)) || 
+     (is_diagonal_move?(stop) && is_space_occupied?(board, stop) && is_opponent?(board, stop))
   end
-end
\ No newline at end of file
+ 
+  private
+
+  def valid_move_by_two?(stop)
+    (is_white? && position_y == 1 && stop[1] == 3 && !is_space_occupied?(board, stop)) || 
```

Comment:
I think this should check if the move is obstructed as well. You could have a piece in front of a pawn and still try to move it two spaces